pyproject.toml

```diff
@@ -13,10 +13,10 @@ einops = "^0.7.0"
 datasets = "^2.0.0"
 dataclasses-json = "^0.6.4"
 jaxtyping = "^0.2.28"
-sae-lens = "^0.5.0"
 transformer-lens = "^1.0.0"
 eindex-callum = "^0.1.0"
 rich = "^13.7.1"
+matplotlib = "^3.8.4"
```

Comment:
why is matplotlib needed?

Reply:
previously pyright failed with:
```
Run poetry run pyright

added 1 package, and audited 2 packages in 2s

found 0 vulnerabilities
/home/runner/work/sae_vis/sae_vis/sae_vis/html_fns.py
  /home/runner/work/sae_vis/sae_vis/sae_vis/html_fns.py:6:6 - error: Import "matplotlib" could not be resolved (reportMissingImports)
1 error, 0 warnings, 0 informations 
Error: Process completed with exit code 1.
```

Reply:
Ah I guess that dependency was previously satisfied by depending on SAELens. Makes sense for that to be a direct dependency if it's used in the project